### PR TITLE
🐛  🏗  Fix `amp lint` being run outside of CI

### PR DESCRIPTION
**Pull request by rileyajones**
I'm not sure when this began but `amp lint` is currently not passing outside of CI due to a handful of old restricted imports.

### CODE REVIEW

**Review by jridgewell — APPROVED** [review]

**Review by rsimha — COMMENTED** [review]
Are you seeing errors or warnings? IIRC, @rcebulko made it so that a subset of warnings are silenced during CI but show up during development so that they can be gradually fixed as code is changed.

**Edit:** Relevant thread: https://github.com/ampproject/amphtml/pull/33016#discussion_r586931446

**Review by rcebulko — COMMENTED** [review]
What do you mean not passing? As in giving a warning or throwing an error?

**Review by rcebulko — CHANGES_REQUESTED** [review]
I think this is the result of a different file being removed, since `fetch.js` polyfill was exempted previously. I'm actively looking into this, could you wait a few while I investigate?

File: src/.eslintrc.js [hunk 1]
```diff
@@ -27,13 +27,19 @@ module.exports = {
...
+              './3p-frame',
+              './url',
+            ],
           },
           {
             // Disallow importing non-core dependencies into polyfills
             'target': 'src/polyfills',
             'from': 'src',
-            'except': ['./core', './polyfills'],
+            'except': ['./core', './polyfills', './utils/log'],
```
Comment (rcebulko) [inline src/.eslintrc.js:42]: This file prevents us from accidentally pulling in large subsystems, ex. importing the entire logging system by mistake in certain areas of code that shouldn't, such as code that may be bundled with Bento components where we don't want all the bytes of a logging system we shouldn't use. Individual files with "illegal" imports, like `#polyfills/fetch` which depends on `#utils/log`, are excluded in the `overrides` section below. They don't do anything in CI, but they give a warning locally. This is WAI, and a fresh `main` checkout and `amp lint` run seems to indicate that all is as it should be and there are no new errors.

**Comment by rileyajones** [comment]
@rcebulko I'm happy to wait your investigation, thanks for looking into this

**Comment by rcebulko** [comment]
> @rcebulko I'm happy to wait your investigation, thanks for looking into this

AFAICT this is working as intended, and `amp lint` doesn't fail, it logs warnings. Are you experiencing something different?

**Comment by rileyajones** [comment]
After an offline discussion with @rcebulko it seems that the root problem behind this is being phased out.

**Pull request closed without merging**